web/src/actions/apd.test.js

```diff
@@ -475,13 +474,7 @@ describe('apd actions', () => {
       const store = mockStore(state);
       fetchMock.onPut('/apds/id-to-update').reply(403, [{ foo: 'bar' }]);
 
-      const expectedActions = [
-        { type: actions.SAVE_APD_REQUEST },
-        {
-          type: notificationActions.ADD_NOTIFICATION,
-          message: 'Save successful!'
```

Comment:
We're still displaying the `Save successful!` message, though, aren't we?

Reply:
Yep. It's handled by the `SaveButton` component now.